app/models/bgeigie_import.rb

```diff
@@ -50,6 +50,12 @@ def self.uploaded_after(time)
     where("created_at > ?", time)
   end
 
+  class << self
```

Comment:
Is there a reason for using `class << self` here instead of `def self.` like the other scoping methods? I'm find with either, but consistency is always good.